src/cares_wrap.cc

```diff
@@ -1005,20 +1547,30 @@ class QuerySoaWrap: public QueryWrap {
 
     Local<Object> soa_record = Object::New(env()->isolate());
 
-    soa_record->Set(env()->nsname_string(),
-                    OneByteString(env()->isolate(), soa_out->nsname));
-    soa_record->Set(env()->hostmaster_string(),
-                    OneByteString(env()->isolate(), soa_out->hostmaster));
-    soa_record->Set(env()->serial_string(),
-                    Integer::New(env()->isolate(), soa_out->serial));
-    soa_record->Set(env()->refresh_string(),
-                    Integer::New(env()->isolate(), soa_out->refresh));
-    soa_record->Set(env()->retry_string(),
-                    Integer::New(env()->isolate(), soa_out->retry));
-    soa_record->Set(env()->expire_string(),
-                    Integer::New(env()->isolate(), soa_out->expire));
-    soa_record->Set(env()->minttl_string(),
-                    Integer::New(env()->isolate(), soa_out->minttl));
+    soa_record->Set(context,
+                    env()->nsname_string(),
+                    OneByteString(env()->isolate(),
+                                  soa_out->nsname)).FromJust();
+    soa_record->Set(context,
+                    env()->hostmaster_string(),
+                    OneByteString(env()->isolate(),
+                                  soa_out->hostmaster)).FromJust();
+    soa_record->Set(context,
+                    env()->serial_string(),
+                    Integer::New(env()->isolate(), soa_out->serial)).FromJust();
+    soa_record->Set(context,
+                    env()->refresh_string(),
+                    Integer::New(env()->isolate(),
+                                 soa_out->refresh)).FromJust();
+    soa_record->Set(context,
+                    env()->retry_string(),
+                    Integer::New(env()->isolate(), soa_out->retry)).FromJust();
+    soa_record->Set(context,
+                    env()->expire_string(),
+                    Integer::New(env()->isolate(), soa_out->expire)).FromJust();
+    soa_record->Set(context,
+                    env()->minttl_string(),
+                    Integer::New(env()->isolate(), soa_out->minttl)).FromJust();
```

Comment:
Fwiw I didn’t expect you do change the `Set()` calls in unrelated code too, but I really appreciate that you put in the work; thank you!